tests/Functional/ImportCommandsTest.php

```diff
@@ -49,11 +91,119 @@ public function testImportFiles()
      * @covers \Pantheon\Terminus\Commands\Import\SiteCommand
      *
      * @group import
-     * @group todo
+     * @group long
+     *
+     * @throws \Exception
+     * @throws \GuzzleHttp\Exception\GuzzleException
      */
-    public function testImportSite()
+    public function testImportSiteCommand()
     {
-        $this->fail('Not implemented. Requirement: a Drupal-based test site to create a site archive via Drush.');
+        $uniqueId = uniqid();
+
+        // Create a site to archive.
+        $this->archivedSiteName = sprintf('site-archive-d7-%s', $uniqueId);
+        $this->terminus(
+            sprintf(
+                'site:create %s %s drupal7 --org=%s',
```

Comment:
We don't need to create the archived site every time. Write a script in /scripts or /tests/bin, and include instructions on how to run it should someone need to re-create the test fixture. Then, make a well-known site in the Pantheon org "CI Fixtures for Projects", and refer to it by name (and path to the archive dump) in the tests via literals or constants. The test can import the same dump file every time.

Reply:
> We don't need to create the archived site every time. Write a script in /scripts or /tests/bin, and include instructions on how to run it should someone need to re-create the test fixture. Then, make a well-known site in the Pantheon org "CI Fixtures for Projects", and refer to it by name (and path to the archive dump) in the tests via literals or constants. The test can import the same dump file every time.

Updated